preplan-backend/controllers/eventController.js

```diff
@@ -373,9 +372,149 @@ router.get('/position/:id', [authJwt.verifyToken, authJwt.isOrganizer], (req, re
         });
 })
 
-router.post('/create/timeSlot', [authJwt.verifyToken, authJwt.isOrganizer], (req, res) => {
+router.get('/edition/:id/registered', [authJwt.verifyToken, authJwt.isOrganizer], (req, res) => {
```

Comment:
would like to keep the term inscription please, as registering is creating an account on the website.

Reply:
Alrighty

Reply:
is this not a method to send any inscription without filters on the status?

Reply:
well this gets all the register volunteers to an event